rollup/rollup-cdn-preview.config.js

```diff
@@ -0,0 +1,12 @@
+import { toCdnEntryName } from '../tasks/cdn-entry-name.js';
+import { CDN_ENVIRONMENTS } from '../tasks/cdn-environments.js';
+import { getCurrentBranchSync } from '../tasks/git-utils.js';
+import { getCdnRollupConfig } from './rollup-cdn-common.js';
+
+export default getCdnRollupConfig(getCdnEntryName(), false);
+
+function getCdnEntryName () {
+  const nameFromEnv = process.env.PREVIEW;
+  const name = (nameFromEnv == null || nameFromEnv.length === 0) ? getCurrentBranchSync() : nameFromEnv;
+  return toCdnEntryName(CDN_ENVIRONMENTS.preview, name);
+}
```

Comment:
suggestion: a bit like for `rollup-cdn-config.js`, I would just inline the `getCdnEntryName` function:

```js
const nameFromEnv = process.env.PREVIEW;
const name = (nameFromEnv == null || nameFromEnv.length === 0) ? getCurrentBranchSync() : nameFromEnv;
const cdnEntryName = toCdnEntryName(CDN_ENVIRONMENTS.preview, name);

export default getCdnRollupConfig(cdnEntryName, false);
```